### PR TITLE
Build script for TensorFlow 1.10.0 with GPU support on PPC64LE

### PR DESCRIPTION
**Pull request by wdirons**
Run from within the docker container: docker.io/nvidia/cuda-ppc64le:9.2-cudnn7-devel-ubuntu16.04
Requires nccl_2.2.13-1+cuda9.2_ppc64le.tgz downloaded from https://developer.nvidia.com/nccl/nccl-download
be in the same directory as the sript

To run this script
apt-get update -y
apt-get install -y python
python build_tensorflow_1_10_0_gpu.py

The resulting wheel file will be in the tensorflow_pkg subdirectory from where the script was invoked.

### CODE REVIEW

**Comment by wdirons** [comment]
@sandipmgiri , @jayfurmanek, @gerrith3 - Please review, I lack authority to add you add reviews. Thank you.

**Comment by gerrith3** [comment]
Priya, can you have Sandip review to make sure this is similar to our normal build scripts?  When he reviews/approves, can you merge this in as well?  Thanks!!

File: tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    for GPU model to Compute Capabilities mapping.
+    '''
+
+    f= open('tensorflow/.bazelrc',"w+")
+    f.write("build --action_env PYTHON_BIN_PATH='/usr/bin/python'\n\
+build --action_env PYTHON_LIB_PATH='/usr/local/lib/python2.7/site-packages'\n\
+build --python_path='/usr/bin/python'\n\
+build --define with_jemalloc=true\n\
+build --define with_hdfs_support=true\n\
+build:gcp --define with_gcp_support=false\n\
```
Comment (jayfurmanek) [inline tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py]: These build options differ slightly than whats enabled by upstream in their community builds. It also differs a bit from the other recipes in build-scripts. Perhaps we should make them common.

**Comment by sandipmgiri** [comment]
Apart from Jason's comment everything else looks good to me.

File: tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    'wget https://github.com/bazelbuild/bazel/releases/download/0.15.0/bazel-0.15.0-dist.zip',
+    'mv bazel-0.15.0-dist.zip bazel/',
+    'unzip -o bazel/bazel-0.15.0-dist.zip -d bazel/',
+    'bazel/compile.sh',
+    'cp bazel/output/bazel /usr/local/bin/'
+    ]
+git = [
+    'rm -rf tensorflow',
+    'git clone -b v1.10.0 https://github.com/tensorflow/tensorflow',
+    'wget  https://github.com/wdirons/tensorflow/commit/03c24d6f13bb30d87745c6eb752019457863b00c.patch',
```
Comment (wdirons) [inline tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py]: I realized this patch is only necessary for RHEL / gcc 4.8. I'm going to remove it for ubuntu. I'll test that when testing the changes for Jason's comment.

File: tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py (hunk 1 above)
Comment (wdirons) [reply tensorflow/Ubuntu/tensorflow_1.10.0_gpu_ubuntu_16.04.py]: I will enable the flags for Google Cloud Platform, Amazon AWS and Apache Kafka. I think that will make things in sync with the upstream defaults.

**Comment by wdirons** [comment]
The requested changes have been made and tested.

**Comment by seth-priya** [comment]
Thanks @sandipmgiri @wdirons @jayfurmanek - merged!

**Pull request merged**